prov/shm/src/smr.h

```diff
@@ -173,7 +175,7 @@ struct smr_cmd_ctx {
 OFI_DECLARE_FREESTACK(struct smr_rx_entry, smr_recv_fs);
 OFI_DECLARE_FREESTACK(struct smr_cmd_ctx, smr_cmd_ctx_fs);
 OFI_DECLARE_FREESTACK(struct smr_tx_entry, smr_pend_fs);
-OFI_DECLARE_FREESTACK(struct smr_sar_entry, smr_sar_fs);
```

Comment:
Not sure what opr is? Can we call this rx_pend_fs or something to indicate that it is to track receive side pending operations? And change the tx one to tx_pend_fs?

Reply:
I renamed it to smr_tx_fs and smr_rx_fs respectively to remain consistent with the smr_ prepended to the other names. I also named the actual fields in the  smr_ep to tx_fs and rx_fs respectively